test/functional/ping_test.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'test_helper'
+
+describe 'ping' do
+  before(:each) do
+    HammerCLIKatello::Command.any_instance.stubs(:send_request).returns('services' => [])
```

Comment:
Have you considered using `api_expects` construct to stub the api call? Something like: https://github.com/Katello/hammer-cli-katello/blob/master/test/functional/host/subscription/unregster_test.rb#L13-L16
It will enable for checking whether the command hits the correct api endpoint.